sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/SyntaxHelper.cs

```diff
@@ -198,5 +198,9 @@ public static Location FindIdentifierLocation(this BaseMethodDeclarationSyntax m
             return identifierSyntax?.GetLocation();
         }
 
+        public static int GetLine(this SyntaxToken token)

```

Comment:
I understand we are returning the value of the Line property, but I think GetLineNumber will be a more meaningful name.